rubyric/02_getting_started.rst

```diff
@@ -24,6 +24,22 @@ a new menu item in A+ choose Edit course and Menu and press Add new menu item.
 You can also choose to use Rubyric through Haka login. In that case you don't
 need to add Rubyric to menu.
 
+.. important::
+
+  If you want to test Rubyric locally, you must read the
+  `Official Rubyric documentation <https://github.com/Aalto-LeTech/rubyric/blob/master/doc/rubyric.md#connect-local-rubyric-to-a-course-in-docker-optional>`_
+  and follow the instructions on how to use a local copy of Rubyric along with A+.
+  It is also important to mention that currently testing Rubyric locally can be
+  slightly difficult. Therefore, we recommend adding the desired `submit`
+  directives to the course along with the corresponding `config.yaml` file,
```

Comment:
Weird language here. "Desired directives", "desired" is not a good word here. "config.yaml should contain the configuration", why "should"? It really contains the configuration, but that was obvious.

Reply:
"only then verify that you can configure": this makes it all sound too complicated and as if you can not set up Rubyric without huge trouble. This should be toned down. The focus should be on describing how to set up Rubyric correctly instead of scaring people away.

Reply:
I reworded most of the changes.